src/pages/MenuPage.ts

```diff
@@ -5,24 +5,44 @@ export class MenuPage extends AbstractPage {
   readonly page: Page;
   readonly url: string;
   readonly coffeeItemLocators: Locator;
+  readonly addToCartConfirmButton: Locator;
+  readonly addToCartRejectButton: Locator;
 
   constructor(page: Page) {
     super(page);
     this.page = page;
     this.url = ''; 
     this.coffeeItemLocators = page.locator(`li:has(h4)`);
+    this.addToCartConfirmButton = page.getByRole('button', { name: 'Yes'});
```

Comment:
Consider simlper `confirmButton` and `rejectButton`;

Reply:
There is a use case when another 'promo' dialog appears on the same page and it also has confirm/reject buttons (containing other names but doing the same action).  The planned locators are:
    this.addToCartConfirmButton = page.getByRole('button', { name: 'Yes'});
    this.addToCartRejectButton = page.getByRole('button', { name: 'No'});
    this.promoConfirmButton = page.getByRole('button', { name: 'Yes, of course!'});
    this.promoRejectButton = page.getByRole('button', { name: 'Nah, I\'ll skip.'});

Let's go back to this topic once I implement relevant tests for promo.